### PR TITLE
Updates to various docs

### PR DESCRIPTION
**Pull request by J-Massey**
These are just a couple of clarifications I think these docs might need.

### CODE REVIEW

**Comment by weymouth** [comment]
@b-fg Can you please look this over?

File: HPC/Iridis4/subLotus [hunk 1]
```diff
@@ -1,17 +1,18 @@
-!/bin/bash -login
+#!/bin/bash -login
+#PBS -S /bin/bash
 
-# Job script to run mpi job in parallel. The line above loads your bash profile into the computational node (required to run Lotus). To run other applications the -login flag is not required.
+# Job script to run mpi job in parallel. The line above loads your bash profile into the computation$
```
Comment (b-fg) [inline HPC/Iridis4/subLotus:4]: What's "computation$" ?

File: HPC/Iridis4/subLotus [hunk 2]
```diff
@@ -23,7 +24,7 @@ module load R/3.3.0
 # Change to directory from which job was submitted
 cd $PBS_O_WORKDIR
 
-# Run Lotus (mpif90). Note that total np must be the same as stated before. You can also use your /scratch
+# Run Lotus (mpif90). Note that total np must be the same as stated before. You can also use your /s$
```
Comment (b-fg) [inline HPC/Iridis4/subLotus:27]: Same here, I guess this is a bash variable for scratch? This is just a comment though, so the more explicit the better ;)

**Comment by b-fg** [comment]
All looks good except the modification of those two comments, which makes it less clear imho. I will revert that and merge the PR. Thank you very much for contributing to the documentation, keep these coming! :)

**Pull request merged**